code401challenges/src/main/java/code401challenges/linkedList/LinkedList.java

```diff
@@ -2,17 +2,17 @@
 
 import java.util.StringJoiner;
 
-public class Stack {
+public class LinkedList {
     public static void main(String[] args){
-        Stack t = new Stack();
+        LinkedList t = new LinkedList();
         t.append(5);
         System.out.println(t);
     }
```

Comment:
We do not want to have a main method like this in a Library class like `LinkedList`